frontend/services/entities/operating_systems.ts

```diff
@@ -78,10 +78,6 @@ export const getOSVersions = ({
     per_page,
   };
 
-  if (teamId !== API_NO_TEAM_ID) {
-    params.team_id = teamId;
-  }
-
```

Comment:
this was added in https://github.com/fleetdm/fleet/pull/17862 I guess as a workaround for the auth error when sending `team_id=0` but it was causing this page to show os versions for "all teams"